docs/guides/migrate2rocky.md

```diff
@@ -69,13 +69,13 @@ Then clone the rocky-tools repository with:
 git clone https://github.com/rocky-linux/rocky-tools.git
 ```
 
-Note: this method will download all of the scripts and files in the rocky-tools repository.
+Note: This method will download all scripts and files from the Rocky Tools repository.
```

Comment:
![rocky-tools](https://github.com/rocky-linux/documentation/assets/4265212/df9942d6-a3e1-41fb-a9a7-f4027c248ee1)
In this case, @gannazhyrnova it is actually "rocky-tools"... (see image)